sdk/src/main/java/com/endare/adhese/sdk/views/AdView.java

```diff
@@ -41,6 +43,7 @@ public class AdView extends WebView {
     private OnViewImpressionNotifiedListener viewImpressionNotifiedListener;
     private OnErrorListener errorListener;
     private OnAdClickListener onAdClickListener;
+    private AdvancedOnAdClickListener advancedOnAdClickListener;
```

Comment:
maybe document if you can have only one of them or both of them ?

Reply:
I've added it to the readme